doc/convert.md

```diff
@@ -1,42 +1,68 @@
+| rfc4175_422be10 | rfc4175_422le8  | &#x2705; |          | &#x2705; |  &#x2705;   |
+| rfc4175_422le10 | v210            | &#x2705; |          | &#x2705; |  &#x2705;   |
+| rfc4175_422le10 | rfc4175_422be10 | &#x2705; |          | &#x2705; |  &#x2705;   |
+| rfc4175_422le10 | yuv422p10le     | &#x2705; |          |          |             |
+| yuv422p10le     | rfc4175_422be10 | &#x2705; |          | &#x2705; |             |
+| yuv422p10le     | rfc4175_422le10 | &#x2705; |          |          |             |
+| v210            | rfc4175_422be10 | &#x2705; |          | &#x2705; |  &#x2705;   |
+| y210            | rfc4175_422be10 | &#x2705; |          | &#x2705; |             |
+| rfc4175_422be12 | yuv422p12le     | &#x2705; |          |          |             |
+| rfc4175_422be12 | rfc4175_422le12 | &#x2705; |          |          |             |
+| rfc4175_422le12 | yuv422p12le     | &#x2705; |          |          |             |
+| rfc4175_422le12 | rfc4175_422be12 | &#x2705; |          |          |             |
+| yuv422p12le     | rfc4175_422be12 | &#x2705; |          |          |             |
+| yuv422p12le     | rfc4175_422le12 | &#x2705; |          |          |             |
+| rfc4175_444be10 | yuv444p10le     | &#x2705; |          |          |             |
+| rfc4175_444be10 | gbrp10le        | &#x2705; |          |          |             |
+| rfc4175_444be10 | rfc4175_444le10 | &#x2705; |          |          |             |
+| rfc4175_444le10 | yuv444p10le     | &#x2705; |          |          |             |
+| rfc4175_444le10 | gbrp10le        | &#x2705; |          |          |             |
+| rfc4175_444le10 | rfc4175_444be10 | &#x2705; |          |          |             |
+| yuv444p10le     | rfc4175_444be10 | &#x2705; |          |          |             |
+| yuv444p10le     | rfc4175_444le10 | &#x2705; |          |          |             |
+| gbrp10le        | rfc4175_444be10 | &#x2705; |          |          |             |
+| gbrp10le        | rfc4175_444le10 | &#x2705; |          |          |             |
+| rfc4175_444be12 | yuv444p12le     | &#x2705; |          |          |             |
+| rfc4175_444be12 | gbrp12le        | &#x2705; |          |          |             |
+| rfc4175_444be12 | rfc4175_444le12 | &#x2705; |          |          |             |
+| rfc4175_444le12 | yuv444p12le     | &#x2705; |          |          |             |
+| rfc4175_444le12 | gbrp12le        | &#x2705; |          |          |             |
+| rfc4175_444le12 | rfc4175_444be12 | &#x2705; |          |          |             |
+| yuv444p12le     | rfc4175_444be12 | &#x2705; |          |          |             |
+| yuv444p12le     | rfc4175_444le12 | &#x2705; |          |          |             |
+| gbrp12le        | rfc4175_444be12 | &#x2705; |          |          |             |
+| gbrp12le        | rfc4175_444le12 | &#x2705; |          |          |             |
```

Comment:
The table is too long now:)
Maybe we can spilt it with 3 tables, one with current 422 10 bit, one with 444 10bit, the last with 444 12 bit.